docs/support/migration-v8.md

```diff
@@ -83,7 +84,10 @@ const trial = {
 ```
 
 The `button_html` parameter can also support different HTML for each button. 
-See the [plugin documentation](https://www.jspsych.org/plugins/jspsych-html-button-response/) for more details.
+See the [plugin documentation](https://www.jspsych.org/latest/plugins/html-button-response/index.html) for more details.
+
+For plugin developers: if you are writing a plugin and updating parameters to use functions, 
```

Comment:
I'm not sure what situation this refers to. Maybe an example would help?

Reply:
I'm also worried that "mock these functions" assumes a lot of knowledge about how to do that.

Reply:
going to remove this for now, i think when we flesh out our plugin development tutorials, we'll go ahead and put that tidbit there.